roxie/ccd/ccdmain.cpp

```diff
@@ -1307,7 +1307,7 @@ int CCD_API roxie_main(int argc, const char *argv[], const char * defaultYaml)
                         }
                         const char *soname =  roxieFarm.queryProp("@so");
                         const char *config  = roxieFarm.queryProp("@config");
-                        Owned<IHpccProtocolPlugin> protocolPlugin = ensureProtocolPlugin(*protocolCtx, soname);
+                        IHpccProtocolPlugin *protocolPlugin = ensureProtocolPlugin(*protocolCtx, soname);
```

Comment:
This will fix the crash, but it isn't the correct fix.  ensureProtocolPlugin is returning inconsistently linked objects.
HPCC-25664 created to fix it properly.